backend-rust/src/graphql_api.rs

```diff
@@ -3346,6 +3559,56 @@ impl SearchResult {
     }
 }
 
+fn decode_value_with_schema(
+    opt_schema: &Option<Type>,
```

Comment:
```suggestion
    opt_schema: Option<&Type>,
```
[See e.g. here.](https://www.reddit.com/r/rust/comments/wqsxk2/is_it_better_to_pass_optiont_or_optiont/ikokviw/)